### DISCUSSION TITLE
Error at update 20211203

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#678] [Fixed Issues]
`./install.sh: line 20: $file: ambiguous redirect`

```
11  # 20211203
12  rm -rf /srv/http/data/embedded
13  mkdir -p $dirshm/{airplay,embedded,spotify,local,online,sampling,webradio}
14  
15  sed -i '/chromium/ d' /etc/pacman.conf
16  
17  file=$( ls /etc/systemd/network/eth* )
18  grep -q RequiredForOnline=no $file || echo "
19  [Link]
20  RequiredForOnline=no" >> $file
```
EDIT: More Infos
The command in line 17
`file=$( ls /etc/systemd/network/eth* )`
find two entries (which are responsible for the "ambiguous redirect" in line 20):
`/etc/systemd/network/eth.network 
/etc/systemd/network/eth0.network`
differing in the "Name" parameter only ("Name=eth*" and "Name=eth0")

**Comment by rern** [comment]
Fixed.
